xlive/Blam/Engine/cseries/cseries.h

```diff
@@ -52,6 +52,10 @@ static_assert(sizeof(datum) == 4);
 #define SHORT_BITS SIZEOF_BITS(int16)
 #define LONG_BITS SIZEOF_BITS(int32)
 
+#if(!__INTEL_LLVM_COMPILER)
+#define MSVC
```

Comment:
this name specifier is quite generic, might be conflicting with existing directives